### ISSUE TITLE
--diff has inconsistent exit code depending on if --recursive is present

### ISSUE THREAD

**Issue by mtlynch** [morishitter/tidify#11]
This is related to #8.

I've confirmed that the exit code is set to non-zero for a single file:

```bash
$ ./node_modules/.bin/scssfmt --version
1.0.3
$ echo "foo {width:50px; }" > /tmp/foo.scss
$ ./node_modules/.bin/scssfmt /tmp/foo.css --diff
/tmp/foo.css
-foo {width:50px; }
+foo { width: 50px; }
$ echo $?
1
```

But if the user uses the `--recursive` flag, the exit code is not set:

```
$ ./node_modules/.bin/scssfmt --version
1.0.3
$ echo "foo {width:50px; }" > /tmp/foo.scss
$ echo "bar {width:50px; }" > /tmp/bar.scss    
$ ./node_modules/.bin/scssfmt --recursive "/tmp/*.scss" --diff
/tmp/bar.scss
-bar {width:50px; }
+bar { width: 50px; }

/tmp/foo.scss
-foo {width:50px; }
+foo { width: 50px; }
$ echo $?
0
```

* **Expected**: With the `--recursive` flag, the exit code is non-zero if there were diffs in any files.
* **Actual**: Passing the `--recursive` flag seems to always make the return code zero.

**Comment by msaktype** [comment]
@mtlynch Thank you for your report. I will fix it soon! :)

**Comment by msaktype** [comment]
@mtlynch Fixed this in https://github.com/morishitter/scssfmt/pull/12 and published v1.0.4. Please update scssfmt :) Thanks.

**Comment by mtlynch** [comment]
Thanks!
